app/actions/views/root.js

```diff
@@ -87,7 +87,7 @@ export function loadFromPushNotification(notification) {
         // we should get the posts
         if (channelId === currentChannelId) {
             dispatch(markChannelAsRead(channelId, null, false));
-            await dispatch(retryGetPostsAction(getPosts(channelId)));
+            await retryGetPostsAction(getPosts(channelId), dispatch, getState);
```

Comment:
Interesting.  I'd like to know why the other one is not working as expected.

Reply:
because the other one is not an action is a function that calls the action